Airports/EGPG/SMR/regions.txt

```diff
@@ -0,0 +1,182 @@
+; Start Cumbernauld (EGPG) Regions
+; - BG
+REGIONNAME Cumbernauld
+488718 N055.58.26.189 W003.58.56.335
```

Comment:
Define this colour in the Colours.txt file please, then replace ```488718``` with the name of the colour